proto/storage/report_configuration.proto

```diff
@@ -35,6 +35,7 @@ message ReportConfiguration {
   repeated NotifierConfiguration notifiers = 12;
   SlimUser creator = 13; // @gotags: sql:"ignore_labels(User ID)"
   int32 version = 14; // version=0 is unmigrated v1 config, version=1 is migrated v1 config and version=2 is v2 config
+  VulnerabilityReportOptionalColumns optionalColumns = 15;
```

Comment:
I think this would make it more difficult to use the report config for a different report type later.  Would probably want to put it in a `oneof` like filters.